holmes/core/conversations_worker/realtime_manager.py

```
@@ -320,16 +310,19 @@ async def _run(self) -> None:
             # so transient startup failures (e.g. Supabase 503) are retried
             # instead of killing the thread.
             while not self._stop_event.is_set():
-                success = await self._full_reconnect()
-                if success:
+                try:
+                    await self._full_reconnect()
                     reconnect_attempts = 0
                     break
-                reconnect_attempts += 1
+                except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)